commands/push.go

```diff
@@ -71,15 +72,15 @@ func pushStack(services *stack.Services, queueDepth int) {
 		go func(index int) {
 			wg.Add(1)
 			for function := range workChannel {
-				fmt.Printf("[%d] > Pushing: %s.\n", index, function.Name)
+				fmt.Printf(aec.YellowF.Apply("[%d] > Pushing %s.\n"), index, function.Name)
 				if len(function.Image) == 0 {
 					fmt.Println("Please provide a valid Image value in the YAML file.")
 				} else {
 					pushImage(function.Image)
 				}
+				fmt.Printf(aec.YellowF.Apply("[%d] < Pushing %s done.\n"), index, function.Name)
 			}
 
-			fmt.Printf("[%d] < Pushing done.\n", index)
```

Comment:
Retain the message please as "Push worker [%d] done"

Reply:
`fmt.Printf(aec.YellowF.Apply("[%d] Push worker done"), index)`
is it's ok with color and index as prefix ?

Reply:
Maybe let's rephrase to "[%d] worker done." for both. Yellow is fine or normal.